src/EventListener/ApiExceptionListener.php

```diff
@@ -63,6 +71,21 @@ public function onKernelException(GetResponseForExceptionEvent $event): void
                 $data['dev'] = $this->getDevBlock($exception);
             }
 
+            $previousData = [];
+            $previousException = $exception->getPrevious();
+
+            if ($previousException) {
+                $previousData = array_merge(
+                    $this->getDataFromException($exception),
+                    ['message' => $previousException->getMessage()]
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "message" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)